products/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.urls import path
+from .views import ProductDetails
+urlpatterns = [
+  path('/details/<products_id>', ProductDetails.as_view())
```

Comment:
restful api 에 따른다면 굳이 지금과 같이 details 라는 단어를 사용하지 않아도 product 의 detail 이라는 것을 알 수 있습니다.
`products/1` 
1 이 없을 때는 products 전부를 가져온다는 의미.
1을 넣으면 products 에서 pk가 1인 product 의 detail 을 가져온다는 의미가 됩니다.
```suggestion
  path('/<products_id>', ProductDetails.as_view())
```

Reply:
숙지하겠습니다!